mods/cnc/rules/defaults.yaml

```diff
@@ -15,27 +15,35 @@
 	RenderSprites:
 
 ^1x1Shape:
+	Entrances:
+		UseOccupiedCellsOffsets: true
 	HitShape:
 		UseTargetableCellsOffsets: true
 		Type: Rectangle
 			TopLeft: -512, -512
 			BottomRight: 512, 512
 
 ^2x1Shape:
+	Entrances:
+		UseOccupiedCellsOffsets: true
```

Comment:
You'll need to duplicate the `TargetableOffsets` overrides on the actors to `EntranceOffsets`.